blog-homepage/src/cards/Card.jsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from "react";
+import Audio from "../component/Audio";
+import PropTypes from "prop-types";
+import classes from "./Card.module.css";
+import bookmarkIcon from "../icons/bookmark.svg";
+import bookmarkIconGreen from "../icons/bookmarkGreen.svg";
+
+function Card({
+  articleLength,
+  audio,
+  author,
+  authorImage,
+  isCardMain,
+  date,
+  description,
+  image,
+  website,
+  memberPreview,
+  title,
+}) {
+  const [isBookmarked, setIsBookmarked] = useState(false);
+
+  const bookmarkHandler = () => {
+    setIsBookmarked((previousState) => !previousState);
+  };
+
+  const openSiteInTab = (url) => {
+    window.open(url, "_blank");
+  };
+
+  return (
+    <div className={isCardMain ? classes.card : classes["card-alt"]}>
+      <img
+        className={isCardMain ? "" : classes["img-alt"]}
+        src={image}
+        alt="main article jpeg"
```

Comment:
make sure to use descriptive alt tags in your code when possible.

Reply:
I wasn't sure about this since there was no alt info in the json file which is why I just hard coded a generic description. I was looking at this homework in the scenario where I would be given random articles to display on the page. I'm thinking for this homework I would just pass in a prop description when calling the Card component since I know the order of the articles to be display.

Reply:
I wasn't sure about this since there was no alt info in the json file which is why I just hard coded a generic description. I was looking at this homework in the scenario where I would be given random articles to display on the page. I'm thinking for this homework I could just pass in a prop description when calling the Card component since I know the order of the articles to be display.

Reply:
I did start this homework with separate components to render the main and lower section and then attempted to make a reusable component and ended with this. Thank for letting me know about scss, I will need to take a look.

Reply:
I updated the Card component to include a better descriptive tag for the images.  I created an object with descriptions and the Card will lookup the description to apply based on the image being rendered.

Reply:
Yeah, I know....just a reminder as I will make sure to beat the drum of accessibility. Feel free to add alt text to the data for this week's or just have something. I'm just being annoying. 😄